packages/ui/cypress/tests/wallet-connect.cy.ts

```diff
@@ -0,0 +1,53 @@
+import { settingsPageUrl } from '../fixtures/landingData'
+import { settingsPage } from '../support/page-objects/settingsPage'
+import { topMenuItems } from '../support/page-objects/topMenuItems'
+import { testAccounts } from '../fixtures/testAccounts'
+
+describe('Wallet Connect', () => {
+  it('can see expected wc state when a wallet is not connected', () => {
+    cy.visit(settingsPageUrl)
+    topMenuItems.connectButton().should('be.visible')
+    settingsPage.wallectConnectAccordion().should('be.visible').click()
+    // ensure elements are disabled when not connected
+    settingsPage
+      .walletConnectAlert()
+      .should('be.visible')
+      .should(
+        'contain.text',
+        'Please first select a Pure or Multisig that you are part of, to be able to use it with WalletConnect'
+      )
+    settingsPage.walletConnectKeyInput().should('be.disabled')
+    settingsPage.connectDappButton().should('be.disabled')
+  })
+
+  it('can see expected wc state when wallet is connected', () => {
+    cy.setupAndVisit({
+      url: settingsPageUrl,
+      extensionConnectionAllowed: true,
+      injectExtensionWithAccounts: [testAccounts['Multisig Member Account 1']]
+    })
+    topMenuItems.settingsButton().click()
```

Comment:
That one shouldn't be needed if you're connected already and visited the settings page.
```suggestion
```

Reply:
Ah yeah originally I was click to get through to the settings page but then changed that approach and forgot to take this out. Thanks